<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fenetre</class>
 <widget class="QMainWindow" name="Fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fenetre</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="afficheGraphe">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="conteneurDroite">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>&amp;Edition</string>
    </property>
    <addaction name="actionAttribut_sommet"/>
    <addaction name="actionAttribut_arc"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOuvrir"/>
   <addaction name="actionEnregistrer"/>
   <addaction name="separator"/>
   <addaction name="actionAttribut_sommet"/>
   <addaction name="actionAttribut_arc"/>
   <addaction name="separator"/>
   <addaction name="actionColorer_Directement"/>
   <addaction name="actionColorer_Heuristique"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="toolTip">
    <string>Importer un document XML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="toolTip">
    <string>Enregistrer le graphe sous forme d'un document XML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAttribut_sommet">
   <property name="text">
    <string>Attribut sommet</string>
   </property>
   <property name="toolTip">
    <string>Modifier ou ajouter un attribut Ã  un sommet</string>
   </property>
  </action>
  <action name="actionAttribut_arc">
   <property name="text">
    <string>Attribut arc</string>
   </property>
   <property name="toolTip">
    <string>Ajouter ou modifier l'attribut d'un rc</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionColorer_Directement">
   <property name="text">
    <string>Colorer Directement</string>
   </property>
  </action>
  <action name="actionColorer_Heuristique">
   <property name="text">
    <string>Colorer Heuristique</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>ajouterSommetModel()</slot>
  <slot>ajouterArcModel()</slot>
 </slots>
</ui>
